addons/stock_account/models/product.py

```diff
@@ -129,6 +126,19 @@ def _compute_value_svl(self):
         remaining.value_svl = 0
         remaining.quantity_svl = 0
 
+    def action_revaluation(self):
```

Comment:
maybe add a category 
```
    # -------------------------------------------------------------------------
    # Actions
    # -------------------------------------------------------------------------
```
just before this methd, similar to what's done in this file